meorg_client/client.py

```diff
@@ -233,8 +254,5 @@ def list_endpoints(self) -> Union[dict, requests.Response]:
         dict or requests.Response
             Response from ME.org.
         """
-        
-        return self._make_request(
-            method='get',
-            endpoint='openapi.json'
-        )
+
+        return self._make_request(method="get", endpoint=endpoints.ENDPOINT_LIST)
```

Comment:
Might be a good idea to use const instead of "get"

Reply:
Good catch, all string methods replaced.